### PR TITLE
Update stop/delete scan to accept multiple ids

### PR DESCRIPTION
**Pull request by oussamaessaji**

### CODE REVIEW

**Review by adnaneserrar — APPROVED** [review]

File: src/ostorlab/serve_app/oxo.py [hunk 1]
```diff
@@ -505,11 +505,12 @@ def mutate(root, info: graphql_base.ResolveInfo, scan_id: int):
...
-            if scan is None:
-                raise graphql.GraphQLError("Scan not found.")
-            local_runtime.LocalRuntime().stop(scan_id=str(scan_id))
-            return StopScanMutation(scan=scan)
+            scans = session.query(models.Scan).filter(models.Scan.id.in_(scan_ids)).all()
+            if len(scans) == 0:
+                raise graphql.GraphQLError("No scan is found.")
+            for scan_id in scan_ids:
+                local_runtime.LocalRuntime().stop(scan_id=str(scan_id))
+            return StopScanMutation(scans=scans)
```
Comment (elyousfi5) [inline src/ostorlab/serve_app/oxo.py:513]: i think the mutation should be renamed to StopScansMutation

Comment (elyousfi5) [reply src/ostorlab/serve_app/oxo.py:513]: + you need to change in RE_OXO as well

**Comment by 3asm** [comment]
Ping @oussamaessaji on PR.

File: src/ostorlab/serve_app/oxo.py (hunk 1 above)
Comment (elyousfi5) [reply src/ostorlab/serve_app/oxo.py:515]: ping @oussamaessaji

Comment (3asm) [reply src/ostorlab/serve_app/oxo.py:515]: +1

**Review by 3asm — CHANGES_REQUESTED** [review]

File: src/ostorlab/serve_app/oxo.py (hunk 1 above)
Comment (oussamaessaji) [reply src/ostorlab/serve_app/oxo.py:515]: Should I update it regardless of the numerous mutations that follow this naming in RE?

Comment (3asm) [reply src/ostorlab/serve_app/oxo.py:515]: PING @oussamaessaji

**Review by 3asm — CHANGES_REQUESTED** [review]

**Review by adnaneserrar — DISMISSED** [review]

**Pull request merged**